src/main/scala/org/wikimedia/analytics/hdfstools/HdfsRsyncExec.scala

```diff
@@ -594,15 +594,27 @@ class HdfsRsyncExec(config: HdfsRsyncConfig) {
     }
 
     /**
-     * Function deleting an existing dst path directory if it is empty
+     * Function deleting an existing dst path directory if it is empty.
+     * Only logs in case of dry-run mode
```

Comment:
I see a `log.debug` when not dry-run mode

Reply:
I've been doing it this way everywhere: log.info in dry-run mode, log.debug otherwise

Reply:
No, I just mean the comment isn't accurate.  It logs in both cases.

Reply:
The comment here isn't accurate!  Other than that this is good to go from me merge away please!